scripts/sysman/virtualFunctionManagement.yml

```diff
@@ -0,0 +1,252 @@
+desc: "Virtual Function Management Extension Version(s)"
+version: "1.9"
+name: $s_vf_management_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Virtual function memory type flags"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_info_mem_type_exp_t
+etors:
+    - name: MEM_TYPE_SYSTEM
+      value: "0"
+      desc: "System memory"
+    - name: MEM_TYPE_DEVICE
+      desc: "Device local memory"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Virtual function utilization flag bit fields"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_info_util_exp_flag_t
+etors:
+    - name: INFO_NONE
+      value: "0"
+      desc: "No info associated with virtual function"
+    - name: INFO_MEM_CPU
+      value: "1"
+      desc: "System memory utilization associated with virtual function"
+    - name: INFO_MEM_GPU
+      desc: "Device memory utilization associated with virtual function"
+      value: "2"
+    - name: INFO_ENGINE
+      desc: 'Engine utilization associated with virtual function'
+      value: "4"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Virtual function management properties"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_exp_properties_t
+base: $s_base_properties_t
+members:
+    - type: $s_pci_address_t
+      name: "address"
+      desc: "[out] Virtual function BDF address"
+    - type: $s_uuid_t
+      name: uuid
+      desc: "[out] universal unique identifier of the device"
+    - type: $s_vf_info_util_flag_exp_t
+      name: "flag"
+      desc: "[out] Bit field of utilization flags available."
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Provides memory utilization values for a virtual function"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_util_mem_exp_t
+base: $s_base_state_t
+members:
+    - type: $s_vf_info_mem_type_exp_t
+      name: "memtype"
+      desc: "[out] Memory type flags."
+    - type: uint64_t
+      name: "free"
+      desc: "[out] Free memory size in bytes."
+    - type: uint64_t
+      name: "size"
+      desc: "[out] Total allocatable memory in bytes."
+    - type: uint64_t
+      name: "timestamp"
+      desc: "[out] Wall clock time from VF when value was sampled."
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Provides engine utilization values for a virtual function"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_util_engine_exp_t
+base: $s_base_state_t
+members:
+    - type: $s_engine_group_t
+      name: "type"
+      desc: "[out] The engine group"
+    - type: uint64_t
+      name: "activeCounterValue"
+      desc: "[out] Represents active counter."
+    - type: uint64_t
+      name: "samplingCounterValue"
```

Comment:
since "timestamp" already has the wall clock time, please share if we need the "samplingCounterValue" ?